internal/windows/shcoredll_windows.go

```diff
@@ -14,6 +14,11 @@ var (
 
 // GetDpiForMonitor https://docs.microsoft.com/en-us/windows/desktop/api/libloaderapi/nf-libloaderapi-getmodulehandlew
 func GetDpiForMonitor(monitor HMONITOR, dpiType int32, dpiX, dpiY *uint32) error {
+	err := getDpiForMonitor.Find()
+	if err != nil {
+		return err
```

Comment:
Oh, and please change this to `return errs.NewWithCause(getDpiForMonitor.Name, err)`

Reply:
Done!